docs/README.md

```diff
@@ -42,6 +42,15 @@ $ docker pull ghcr.io/riverqueue/riverui:latest
 $ docker run -p 8080:8080 --env DATABASE_URL ghcr.io/riverqueue/riverui:latest
 ```
 
+### Environment variables
+
+- `DATABASE_URL=...` - define database url
+- `PORT=8080` - define listening port
+- `RIVER_DEBUG=true` - enable debugging logs
+- `CORS_ORIGINS=url1,url2` - define allowed CORS origins
+- `OTEL_ENABLED=true` - enable OTEL integration
+- `BASIC_AUTH_USER=admin`, `BASIC_AUTH_PASSWORD=changeme` - enable basic auth username/password
```

Comment:
Good idea on some basic documentation for this stuff.

Can you sort this list? Better if there's some sort of scheme in how things like this are organized.